pack3d/bvh.go

```diff
@@ -4,14 +4,14 @@ import "github.com/fogleman/fauxgl"
 
 type Tree []fauxgl.Box
 
-func NewTreeForMesh(mesh *fauxgl.Mesh, depth int) Tree {
+func NewTreeForMesh(mesh *fauxgl.Mesh, depth int, space float64) Tree {
```

Comment:
maybe a better name is min_spacing ? or add a comment '/*space is the min required distance between objects*/